build/assets/tuned/openshift-node-performance

```diff
@@ -25,8 +25,10 @@ governor=performance                          #  latency-performance
 energy_perf_bias=performance                  #  latency-performance 
 min_perf_pct=100                              #  latency-performance 
 
-[service]
-service.stalld=start,enable
+# Do not install and enable the stalld service until bugs https://bugzilla.redhat.com/show_bug.cgi?id=1912118 and
+# https://bugzilla.redhat.com/show_bug.cgi?id=1903302 will be fixed
```

Comment:
Please install it and just keep it disabled. That way both us and customers can easily enable it manually to see what happens (systemctl enable...)

Reply:
done